### DISCUSSION TITLE
Does VSCode have any info styles indeed?

### DISCUSSION THREAD

**Discussion by sudoaugustin** [microsoft/vscode-discussions#17] [Extension Development QnA] [answered]
I'm currently developing an extension to customize the vscode theme with UI.

`inputValidation.infoBackground` `editorMarkerNavigationInfo.headerBackground` `editorMarkerNavigationInfo.background` `editorInfo.background`

### But I don't know in which scenario the **info** variants applied to the specific elements.

**Comment by daviddossett** [comment, marked as answer]
@sudoaugustin to clarify, is the question when and where those info theme colors would be used?

For `inputValidation.infoBackground` you can find examples in a read-only editor when trying to type:

<img width="316" alt="CleanShot 2022-06-03 at 10 14 05@2x" src="https://user-images.githubusercontent.com/25163139/171913559-2ec5f98f-849a-42f9-ae41-5970517b2fb8.png">

Or below a text input when there is some supporting context. This is an error in the picture, but there is an info (blue in most default themes) state as well.

https://code.visualstudio.com/api/references/theme-color#input-control

<img width="786" alt="CleanShot 2022-06-03 at 10 26 15@2x" src="https://user-images.githubusercontent.com/25163139/171915343-78d37eeb-db8e-47ad-9967-dc428f790b92.png">


For `editorMarkerNavigationInfo.headerBackground`, `editorMarkerNavigationInfo.background`, and `editorInfo.background`:

https://code.visualstudio.com/api/references/theme-color#editor-widget-colors

<img width="778" alt="CleanShot 2022-06-03 at 10 25 08@2x" src="https://user-images.githubusercontent.com/25163139/171915206-ad6fdb87-0a71-4cb4-b794-161b8ecd4b3e.png">

e.g.

<img width="892" alt="CleanShot 2022-06-03 at 10 29 51@2x" src="https://user-images.githubusercontent.com/25163139/171916124-32220284-26db-44cf-82f7-bca89fb51583.png">

<img width="778" alt="CleanShot 2022-06-03 at 10 25 23@2x" src="https://user-images.githubusercontent.com/25163139/171915234-c14b6019-5b02-431a-af81-471f4fe095b0.png">

**Reply by sudoaugustin** [reply]
Thx u so much
